.github/workflows/cppcheck-daily.yaml

```diff
@@ -18,6 +18,10 @@ jobs:
         run: |
           sudo snap install cppcheck
 
+      - name: Setup problem matchers for cppcheck
+        run: echo "::add-matcher::.github/cppcheck-problem-matcher.json"
+
+      # cspell: ignore suppr
```

Comment:
How the problem matcher works for -daily workflow? How is it shown to users?

Reply:
You don't have the file diff view for -dialy workflow.
But you can check matched errors in workflow summary view.
https://github.com/autowarefoundation/autoware.universe/actions/runs/9763078681?pr=7794
![image](https://github.com/autowarefoundation/autoware.universe/assets/12395284/1ab84b0d-9014-48ac-8d7c-6648b4a71f90)

Reply:
Testing in https://github.com/HansRobo/autoware.universe/actions/runs/9769570736

Reply:
Understood!

One concern is that cppcheck-daily ditects around 8,000 warnings.
Are those warnings all displayed in summary view?

Reply:
Oh, sorry.
The output format in -dialy workflow is different from format in -differential workflow.
I found that it cannot be applied as it is, so I will delete it in -dialy workflow for now.

Reply:
```suggestion
      # cspell: ignore suppr
```